superset/db_engine_specs/base.py

```diff
@@ -476,7 +476,11 @@ def handle_cursor(cls, cursor, query, session):
         pass
 
     @classmethod
-    def extract_error_message(cls, e: Exception) -> str:
+    def extract_error_message(cls, e: Exception):
```

Comment:
pls add a return type here! `-> str`

Reply:
I added a return type for `extract_error_message` but left all the private `_extract_error_message`s without a return type. Mostly because I wanted to avoid putting in a `# noqa` (ugh dict.get). But also, since `extract_error_message` string formats the result of `_extract_error_message`, the return type of `_extract_error_message` shouldn't matter.